src/account/utils.ts

```diff
@@ -77,3 +78,24 @@ export async function mapActorToExternalAccountData(
 export function getAccountHandle(host?: string, username?: string) {
     return `@${username || 'unknown'}@${host || 'unknown'}`;
 }
+
+/**
+ * Convert an account type to an account entity
+ *
+ * @param account Account type
+ */
+export function asAccountEntity(account: AccountType): Account {
+    return new Account(
+        account.id,
+        account.uuid,
+        account.username,
+        account.name,
+        account.bio,
+        account.avatar_url ? new URL(account.avatar_url) : null,
+        account.banner_image_url ? new URL(account.banner_image_url) : null,
+        null, // site
```

Comment:
`AccountType` doesn't have the site data, so we should in theory be safe to leave this as null as anywhere that is using `AccountType` currently won't be accessing the site. We could pass the site as another argument to the fn if needed?